.ci/pull-request-config.yaml

```
@@ -32,14 +32,14 @@ build:
   - project: kiegroup/kogito-apps
     build-command: 
       current: |
-        mvn -f kogito-apps/pom.xml clean install ${{ env.BUILD_MVN_OPTS }}
+        mvn -f kogito-apps/pom.xml clean install -Pvalidate-formatting ${{ env.BUILD_MVN_OPTS }}
     clone:
       - kogito-apps
 
   - project: kiegroup/kogito-examples
     build-command: 
       current: |
-        mvn -f kogito-examples/pom.xml clean install ${{ env.BUILD_MVN_OPTS }}
+        mvn -f kogito-examples/pom.xml clean install -Pvalidate-formatting ${{ env.BUILD_MVN_OPTS }}
```

Can you please use [`validate-formatting` property](https://github.com/kiegroup/kogito-runtimes/blob/master/kogito-build/kogito-build-parent/pom.xml#L1358) to enable the profile? It is more flexible because it let other default profiles to be enabled
```suggestion
        mvn -f kogito-examples/pom.xml clean install -Dvalidate-formatting ${{ env.BUILD_MVN_OPTS }}
```